### PR TITLE
Add image normalize

### PR DESCRIPTION
**Pull request by chakravarthi589**
@ajpmaclean : Please review.

### CODE REVIEW

File: src/Java/ImageData/ImageNormalize.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    // (xmin, ymin, xmax, ymax)
+    double leftViewport[] = new double[] {0.0, 0.0, 0.5, 1.0};
+    double rightViewport[] = new double[] {0.5, 0.0, 1.0, 1.0};
+    
+    //  Setup both renderers
+    vtkRenderer leftRenderer = new vtkRenderer();
+    renWin.AddRenderer(leftRenderer);
+    leftRenderer.SetViewport(leftViewport);
+    leftRenderer.SetBackground(.6, .5, .4);
+
```
Comment (ajpmaclean) [inline src/Java/ImageData/ImageNormalize.java]: Once again, use `vtkNamedColors`, it is bad practise to do what is done here, and the guidelines also emphasise using `vtkNamedColors`. The old C++ examples were made before `vtkNamedColors` existed, and are gradually being updated. The simplest way in this case is:
```java
double left_bkg[] = new double[]{0.6, 0.5, 0.4, 1.0};
double right_bkg[] = new double[]{0.4, 0.5, 0.6, 1.0};
```
If you do this you won't have to update the image.

Comment (chakravarthi589) [reply src/Java/ImageData/ImageNormalize.java]: Changes made as suggested.

Comment (chakravarthi589) [reply src/Java/ImageData/ImageNormalize.java]: As i'll try to keep things as same as C++ program, I tend to do the as it is.
Now on i'll do appropriate updates required. 
Please review now.

Comment (ajpmaclean) [reply src/Java/ImageData/ImageNormalize.java]: Yes thanks. We're trying to make the examples  look as modern as possible. Some of the  C++ predates the introduction of vtkNamedColors - it was specifically written so users could simply name colors from a predefined pallete or create their own.

**Comment by ajpmaclean** [comment]
Thanks

**Pull request merged**